Bibliotheque nationale de France.js

```diff
@@ -390,7 +390,24 @@ var BnfClass = function() {
 		}
 	};
 
-
+	function getCote(record, item) {
+		item.callNumber="";		
+		 var coteTag = record.getFieldSubfields("930");
+		
+		if(coteTag.length) {
+								
+			/*
+			var i;
+			for(i=0; i<coteTag.length-1; i++) {
+				item.callNumber += coteTag[i]['a'] + ";";
+			}	
+			item.callNumber += coteTag[i]['a'];
+			*/
+			item.callNumber += coteTag[0]['c'] + "-" + coteTag[0]['a'];
```

Comment:
Delete what you don't need here. Then you may also can simplify the function more (set `item.callNumber` directly instead of appending to the empty string). Maybe, it is then easier to do the things directly without a special function.